docs/tools/vega-wallet/cli-wallet/latest/guides/build-send-commands.md

```diff
@@ -0,0 +1,234 @@
+</Tabs>
+
+Customise the number of retries (default to 5) using:
+
+<Tabs groupId="operating-systems">
+<TabItem value="windows" label="Windows">
+
+```bash
+vegawallet command send --retries 10 ...
+```
+</TabItem>
+<TabItem value="mac" label="MacOS">
+
+```bash
+./vegawallet command send --retries 10 ...
+```
+</TabItem>
+<TabItem value="linux" label="Linux">
+
+```bash
+./vegawallet command send --retries 10 ...
+
+```
+</TabItem>
+</Tabs>
+
+If you don't want to rely on the nodes defined in the network configuration, you can specify a node address:
+
+<Tabs groupId="operating-systems">
+<TabItem value="windows" label="Windows">
+
+```bash
+vegawallet command send --node-address "ADDRESS"
+```
+</TabItem>
+<TabItem value="mac" label="MacOS">
+```bash
+./vegawallet command send --node-address "ADDRESS"
+```
+</TabItem>
+
+<TabItem value="linux" label="Linux">
+
+```bash
+./vegawallet command send --node-address "ADDRESS"
+```
+</TabItem>
+</Tabs>
+
+See more options by using the help flag:
+
+<Tabs groupId="operating-systems">
+<TabItem value="windows" label="Windows">
+
+```bash
+vegawallet command send --help
+```
+</TabItem>
+<TabItem value="mac" label="MacOS">
+```bash
+./vegawallet command send --help
+```
+</TabItem>
+
+<TabItem value="linux" label="Linux">
+
+```bash
+./vegawallet command send --help
+```
+</TabItem>
+</Tabs>
+
+
+### Create transactions with air-gapped wallets
+
+This is useful for validators that are signing validator commands using their root keys.
+
+The workflow is:
+1. Build and sign a Vega command on an air-gapped computer to get a base64 encoded transaction
+2. Send that transaction using an online computer
+
+
+#### 1. Build a transaction on an air-gapped computer
+
+This will build a transaction containing the specified command, its signature and additional protocol related data:
+
+```bash
+vegawallet command sign --wallet WALLET --pubkey PUBKEY --tx-height TX_HEIGHT COMMAND
+```
+
+`TX_HEIGHT` should be should be close to the current block height when the transaction is applied, with a threshold of approximately 150 blocks.
+
+The resulting transaction is encoded using base64.
+
+You can decode it using `base64` command line utility. First save the transaction in a file `result.txt`, then:
+```bash
+base64 --decode --input result.txt
+```
+
+The transaction will be decoded and displayed on screen.
+
+#### 2. Send a transaction with an online computer
+
+Use any way to transfer the transaction from your air-grapped computer to the online one.
+
+Then, send the transaction using:
+
+```bash
+vegawallet tx send --network NETWORK BASE64_TRANSACTION
+```
+
+:::info
+You will need to respect the block height that you set in you transaction with the command `vegawallet command sign`.
+
+If the block height you set is higher than the blockchain's block height, you should wait until they are more closely aligned - within 150 blocks. Transactions set with a future block height beyond that tolerance will be rejected.
```

Comment:
No, this is wrong. Transactions with a block height in the future are automatically rejected. The tolerance threshold in only used in block height in the past.

If the current height is 200:
* a transaction with block height 10 gets rejected because 200 - 10 = 190 and 190 > 150
* a transaction with block height 60 gets accepted because 200 - 60 = 140 and 140 < 150
* a transaction with block height 201 get rejected because 200 - 201 = -1 and -1 is futuristic.

Reply:
Got it! Fixed but didn't add in your example yet, which I will now, as it's helpful.